docs/guides/query-kyc-state-on-c-chain.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_position: 45
+title: Query KYC State on C-Chain
+description: This document describes how to use Remix IDE to query KYC State of an address on C-Chain
+---
+
+# How to Query KYC State on C-Chain
+
+This document provides a guide on how to use Remix IDE to query the KYC state of
+an address on C-Chain. Follow the steps below to accomplish this.
+
+## Requirements
+
+- MetaMask installation and functionality
+
+## Add Camino Network to MetaMask
+
+To query the KYC state of an address using the Remix IDE, we will use the MetaMask wallet.
+
+We need to add a custom network to MetaMask. In this document, we will use the Columbus test network.
+
+- Click **Add network** from MetaMask.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/0-add-network-to-metamask.png"/>
+<figcaption align="center"><b>Fig.1:</b> Click "Add network"</figcaption>
+</figure>
+
+- On the next screen, click **Add a network manually**
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/1-add-custom-network-to-mm.png"/>
+<figcaption align="center"><b>Fig.2:</b> Click "Add a network manually"</figcaption>
+</figure>
+
+- **Fill in the text fields & click Save**. You can obtain this information from the [MetaMask RPC Endpoints](/guides/metamask-rpc-endpoints.md) page.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/2-enter-details-into-the-network.png"/>
+<figcaption align="center"><b>Fig.3:</b> Fill out the text fields</figcaption>
+</figure>
+
+- Verify that the custom network has been added successfully.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/3-add-custom-network-success.png"/>
+<figcaption align="center"><b>Fig.4:</b> Verify success</figcaption>
+</figure>
+
+## Connecting MetaMask to the Remix IDE
+
+Follow the steps below to connect MetaMask to the IDE:
+
+- Open the Remix IDE web page: https://remix.ethereum.org/
+- Click **Deploy and run transaction** on the right sidebar.
+- Select **Inject Provider - MetaMask** from the **Environment** dropdown menu.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/4-select-injected-provider.png"/>
+<figcaption align="center"><b>Fig.5:</b> Click "Deploy and run transactions" tab and select "Inject Provider" from "Environment"</figcaption>
+</figure>
+
+- Verify that the MetaMask is connected to the Remix IDE.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/5-check-network-connected.png"/>
+<figcaption align="center"><b>Fig.6:</b> Verify network is connected</figcaption>
+</figure>
+
+## Downloading the CaminoAdmin.abi file
+
+We can either compile the contract or use the precompiled ABI file. In this guide, we will use the ABI file.
```

Comment:
@jax what would be the main difference process-wise if the developer was to not use the precompiled ABI file?